ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Impl/Types.hs

```diff
@@ -135,6 +136,11 @@ data TraceEvent blk =
   | TraceCacheEvent !TraceCacheEvent
   deriving (Eq, Generic, Show)
 
+data TraceChunkValidation validateTo =
+    StartedValidatingChunk ChunkNo validateTo
```

Comment:
Would it make sense to comment on what `validateTo` means? I guess we also expect `ChunkNo` to be smaller equal than `validateTo`. Does it make sense to make this data opaque and check that invariant?